jsx/LanguageMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import {useEffect, useState} from 'react';
+import {useTranslation} from 'react-i18next';
+import 'i18n';
+
+const supportedLanguageCodes = ['en', 'fr'];
+
+function filterSupportedLanguageCode(languageCode: string) {
+  if (supportedLanguageCodes.includes(languageCode)) {
+      return languageCode;
+  }
+
+  console.log(`Language '${languageCode}' is not supported by LORIS.`);
```

Comment:
This should either be a warning or removed (both because it's a bad situation to be in and because console.log will cause the static analysis tests to fail.)

Reply:
I'll make that a warning then IMO.